tetraka/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
+
+services:
+  frontend:
+    container_name: tetraka-frontend
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - '.:/app'
+      - '/app/node_modules'
+    ports:
+      - 5000:5000
+    environment:
+      - CHOKIDAR_USEPOLLING=true
```

Comment:
see if getting rid of this makes it still work?

Reply:
Still works.